.coderabbit.yaml

```
@@ -0,0 +1,165 @@
+review_profile: chill 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)